nuts-network/run-tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
-
-set -e # make script fail if any of the tests returns a non-zero exit code
-
+set -o errexit
+set -o nounset
+set -o pipefail
```

Comment:
I think the `set -o errexit` (maybe all of them) should be dropped from the run-test**s**.sh files.

A single failing test should not terminate all e2e tests